README.md

```diff
@@ -1,12 +1,56 @@
 # bikes-for-refugees
-Exercise for HTML lesson 1
+Exercise for HTML & CSS - lesson 1
 
-Fork the repository to your personal Github account and use the following terminal commands to download the files to your projects folder.
+### Set up
 
-`cd ~`
+**Fork** this repository to your personal Github account (look for the 'fork' button on this page).
+
+Then, **clone** it to your machine, using the following terminal commands.
+
+>Go to the folder you wish to save the project in:
 `cd <your-project-directory>`
+
+>Clone the repo:
 `git clone git@github.com:<your_username>/bikes-for-refugees.git`
 
-**Note:**
-Some of the links to images are broken. The correct images have been added in the `images` folder - please replace any broken image links with paths to the images that are in the `images` folder.
+Now, **create a new branch** with your name in the title
+>`git checkout -b bikes-for-refugees/yourname`
+
+You're now ready to code!
+
+### Exercise
+
+The aim of the exercise is for your webpage to look like this: 
+https://codeyourfuture.github.io/syllabus-london/html-css/assets/screenshot-complete.png
```

Comment:
Probably out of scope for this PR, but I wonder if we could deploy a completed version somewhere. Perhaps github pages might be good for that?

Reply:
Yeah but we don't want them inspecting the code now do we ;)

Reply:
Yeah I guess that's a tradeoff. Github pages would hide it under the `gh-pages` branch I believe